src/applications/caregivers/components/FormFields/FacilitySearch.jsx

```diff
@@ -35,6 +36,14 @@ const FacilitySearch = props => {
     return facilities?.length < pagination.totalEntries;
   };
 
+  const ariaLiveMessage = () => {
+    if (newFacilitiesCount > 0) {
+      const facilitySuffix = newFacilitiesCount === 1 ? 'y' : 'ies';
+      return `${newFacilitiesCount} new facilit${facilitySuffix} loaded`;
```

Comment:
Added helper to return either `1 new facility loaded`, `x new facilities loaded` or it returns an empty string.